packages/roosterjs-editor-plugins/lib/plugins/ImageSelection/ImageSelection.ts

```diff
@@ -47,6 +54,11 @@ export default class ImageSelection implements EditorPlugin {
                     const selection = this.editor.getSelectionRangeEx();
                     if (selection.type == SelectionRangeTypes.ImageSelection) {
                         this.editor.select(selection.image);
+                        this.triggerSelectionEvent(
```

Comment:
Let editor core API to trigger this event instead. So that any other plugin can also call that API to trigger the same event

Reply:
do you mean the selectImage Api?

Reply:
yes